.circleci/config.yml

```diff
@@ -41,10 +44,12 @@ anchors:
 
 jobs:
   build:
-    docker:
-      - image: circleci/node:12.14-browsers
+    executor:
+      name: node/default
     steps:
       - checkout
+      - node/install:
+          install-yarn: true
```

Comment:
This will install the `node` version that's in `.nvmrc`.

So we shouldn't have to update this file on bumping the `node` version.